app/views/hiring_staff/identifications/new.html.haml

```diff
@@ -4,7 +4,7 @@
 .govuk-grid-row
   .govuk-grid-column-two-thirds
     - if ENV['FEATURE_SIGN_IN_ALERT'].to_s == "true"
-      .govuk-error-summary{ aria: { labelledby: "error-summary-title" }, role: "alert", tabindex: "-1" }
+      .govuk-error-summary{ aria: { labelledby: "govuk-error-summary-title" }, role: "alert", tabindex: "-1" }
```

Comment:
This should probably be `.govuk-notification.govuk-notification--danger`, but check with UX